flasc/data_processing/dataframe_manipulations.py

```diff
@@ -40,17 +59,48 @@ def filter_df_by_wd(df, wd_range):
 
 
 def filter_df_by_ti(df, ti_range):
+    """Filter a dataframe by turbulence intensity range.
+
+    Args:
+        df (pd.DataFrame): Dataframe with measurements.
+        ti_range ([float, float]): Turbulence intensity range [lower bound, upper bound].
+
+    Returns:
+        pd.DataFrame: Filtered dataframe.
+    """
     df = df[df["ti"] >= ti_range[0]]
     df = df[df["ti"] < ti_range[1]]
     return df
 
 
+# TODO: This function just wraps the other, is it needed?  Maybe to avoid
+# breaking changes?
```

Comment:
Perhaps just here for convenience. Agree that we shouldn't remove it, in case some users are currently using it.

Reply:
Ok, I'll remove the TODO then